README.md

```diff
@@ -31,3 +31,39 @@
 - [Traffic Light](./examples/traffic-light)
 - [Sokoban](./examples/sokoban) -- A type of transport puzzle, in which the player pushes boxes or crates around in a warehouse, trying to get them to storage locations.
 - [Drag-n-Drop](./examples/drag-n-drop) -- A simple drag-n-drop reorderable list.
+
+## Developing
+
+### Requirements
+
+* Latest stable version of [Node](https://github.com/creationix/nvm)
+* Latest stable version of [Yarn](https://yarnpkg.com)
+
+### For generating changelog (for those with NPM publishing rights)
+
+* [Ruby >= v2.1.0](https://www.ruby-lang.org/en/downloads/)
+* [Github Personal Access Token](https://github.com/settings/tokens)
+  - Save the access token you generated for later usage
+
+In your terminal, run the following:
+
+NOTE: `sudo` may be required for all of the following terminal commands -- based
+on where you installed `Ruby`.
+
+```sh
+# If you're on a Debian-based Linux Distribution, install rake
+gem install rake
+
+# Install Github Changelog Generator
+gem install github_changelog_generator
+```
+
+In your `.bash_profile`, `.zshrc` or equivalent shell configuration file add the 
+following:
+
+```sh
+export CHANGELOG_GITHUB_TOKEN=$GITHUB_PERSONAL_ACCESS_TOKEN_GENERATED_ABOVE
+```
+
+and save. In a new terminal you will now be able to successfully run `yarn changelog`
```

Comment:
Remove "and save." We are not dealing with noob computer users who will generate change logs.

Correct:

> In a new terminal, you should now be able to successfully run `yarn changelog`